src/components/aeIdentity/aeIdentity.vue

```diff
@@ -1,32 +1,42 @@
 <template>
-<div :class="{ 'identity-wrap' : true}">
+  <div :class="{ 'identity-wrap' : true}">
 		<div @click="$emit('click', $event)" :class="classObject">
       <div class="flex-row">
         <ae-identity-avatar :address='identity ? identity.address : ""'/>
         <span class="identity-info _short" v-if="collapsed">{{shortAddress}}</span>
 
         <div>
-          <div v-if='tokenAmount' :class="{balance: true, '_small-font': collapsed}">
+          <div v-if='tokenAmount' :class="{balance: true}">
             <span class="amount">{{tokenAmount}}</span>
             <span class="currency-symbol">AE</span>
           </div>
-          <div :class="{balance: true, '_small-font': tokenAmount}">
+          <div :class="{balance: true}">
             <span class="amount">{{amount}}</span>
             <span class="currency-symbol">ETH</span>
           </div>
         </div>
       </div>
-      <div v-if="!collapsed">
-        <div class="public-id-title">Public identity</div>
-        <div class="identity-info _long">{{address}}</div>
+      <div v-if="!collapsed" class="chunked-address">
+        <!-- <div class="identity-info _long">{{address}}</div> -->
+        <div v-for="chunk in chunkAddress" class="chunk">
+          {{chunk}}
+        </div>
       </div>
-      <div class="bottom-inserted-elements" v-if="hasSlot && !collapsed">
-				<slot></slot>
```

Comment:
Will be better to pass buttons through slots, because there can be [other buttons set](https://app.zeplin.io/project/59e4cf99fc0bb2f99a89551b/screen/5a5f16e1f080c170d9e73434), [qr code](https://app.zeplin.io/project/59e4cf99fc0bb2f99a89551b/screen/5a3004ef20b1355a5469a7a7), and even [emptiness](https://app.zeplin.io/project/59e4cf99fc0bb2f99a89551b/screen/5a27ffe6b1549395c783c17b).

Reply:
I actually thought about that. But was thinking it would actually easier to control both the design and bottom functionality from the component itself.

We can set display modes for each bottom if necessary.

Reply:
No, it is not easier. For each bottom part, AeIdentity should define additional props and events.
For example, you added `toggleActive`, `edit`, `copyAddress` events, if you won't remove the slot in AeIdentity, a programmer must know just AeButton api.
The design of bottom functionality also can be controlled outside of AeIdentity.